### DISCUSSION TITLE
UVtools-What is "a file you've already printed"?

### DISCUSSION THREAD

**Discussion by browseruk-xoxy** [sn4k3/UVtools#652] [Q&A] [answered]
Does that mean:

1. The .stl file I feed into my slicer
2. the .fdg that comes out of the slicer for input to my SLA (mLCD) printer
3. Other?

And why does the file need to be one I've already printed?

(Alternatively, is there a commercial product that has the "multiple-timings-calibration-print" facilty; and is clearly documented?)

**Comment by sn4k3** [comment, marked as answer]
A file you already printed in your resin printer. Which is the final file that you feed into printer and press Print. In your case should be the .fdg

FDG does not support multiple times per height

**Reply by browseruk-xoxy** [reply]
Do you guys know if a Voxalab Proxima 6 can accept any other filetype than .fdg?

If so, are any of them compatible with "multiple-timings-calibration-print"? And what slicer is required to produce them?

**Reply by sn4k3** [reply]
> Do you guys know if a Voxalab Proxima 6 can accept any other filetype than .fdg?

It only takes .fdg

**Reply by browseruk-xoxy** [reply]
The (my) confusion arises because: the file does not need to be one I have printed before; it simply needs to be a file that would print if supplied to my mLCD printer.

Here's the (my) problem (with the documenation as it reads). 

I bought the printer and it came with 3 bottles of the manufacturers own-brand resin. I've printed many things with this resin using the default settings  successfully. A few failures here arnd there, but all can be traced back to too thin sections, or badly oriented slicing, or in sufficient support.

When the voxalab resin ran out I purchased and different brand as I wanted something less brittle. I chose to try Amazon's Eono brand "abs-like" resin. (Amazon own brands are usually reasonable quality and reasonable value).

Of the 6 things I have tried to print, 5 have failed. The setting (taken from the bottle label) are obviously not right for my printer.

So, I look around for a way to tune the settings, and find UVtools. I also look for an economical and fast (low Z-height) test object. I found several. But I've atempted to print 3 of them and they all failed. 

Use UVtools to try a spread of settings; but Oh! It needs me to use a file _I've already printed_. What to do?

Please be precise with your intructions and requirements! "Dumbing it down" for the masses does no one any favours.

**Reply by sn4k3** [reply]
Using stock or recommend values by labels or other people are big mistake. Can be took as a starting "base" but should never use them for serious prints, instead you need to find your optimal settings given your environment which is different from all others, even same brand resin with other batch may require to find the optimal exposure. mSLA is not so easy for people without background in the area and them lot of failure comes from not optimized values and improper preparation and use.

> It needs me to use a file I've already printed. What to do?

The instructions are clear but sometimes people make it hard. UVtools takes all information from your file, so why the "file I've already printed"? Because if you use a file that you already print (preferable) with success, UVtools fetch all that settings and use it as stock and then it will require less tune from your part. You can change them in dialog, but at least it will populate those with "sane" values for your printer env. So, what if you took someone file and use it as dummy file? It will populate with values from other user which may not be optimal for you, and if you don't change the settings it can fail. So, in the end, a file "you already print" should be safer to use than a random file that come with printer, that's why is told that way. Of course, nothing prevents you from using any compatible file with your printer but it can put more work if you then have to change all settings after all.

**Reply by browseruk-xoxy** [reply]
"Using stock or recommend values by labels or other people are big mistake." 

Which is why I'm trying to use UVTools to tune the settings....

" lot of failure comes from not optimized values and improper preparation and use."

Which is why I'm trying to use UVTools....

"The instructions are clear but sometimes people make it hard."

Keep telling yourself that; you might end up believing it.

Seems that as .fdg cannot do multiple timings; UVtools doesn't help me anyway...

**Reply by sn4k3** [reply]
FDG is not compatible with multiple exposure and other advanced functions. A list of printers can be seen [here](https://github.com/sn4k3/UVtools/wiki/Printer-compability-with-per-layer-settings-and-advanced-tools)
Still you can use the traditional method (1 test per print) but in that way is better to slice the calibration models than use the UVtools
